docs/getting-started/tutorial-sample-query.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[tutorial-sample-query]]
+=== Querying the data
+
+You can also enter an {es} query to narrow the view of the data.  {kib} supports the 
+<<lucene-query>>.
+
```

Comment:
I think this is a bit misleading. You can use the ES query syntax, or Lucene syntax, (Or kuery syntax if you enable it.) Somewhere, we should talk about the different ways to write queries--not sure if that belongs here, though.